server/src/__test__/products.test.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-undef */
+// eslint-disable-next-line import/no-extraneous-dependencies
+import supertest from 'supertest';
+import sequelize from '../database/config/connection';
+import app from '../app';
+import sync from '../database/sync';
+import buildFakeData from '../database/fakeData/buildFakeData';
+
+beforeAll(async () => {
+  await sync();
+  await buildFakeData();
+});
+
+describe('POST /api/v1/auth/admin/product', () => {
+  test('success add product', (done) => {
+    supertest(app)
+      .post('/api/v1/auth/admin/product')
+      .set('Cookie', ['token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiaXNBZG1pbiI6ZmFsc2UsIm5hbWUiOiJJYnJhaGltIiwiaWF0IjoxNjUwNjY5MDk1fQ.7lzrri5_70Xlm5djI-bu9HPSI7yTnT-yP813FNXZOBk'])
```

Comment:
you can store the token in the `.env` file and use it like this it's more readable and secure
```suggestion
      .set('Cookie', [`token=${proccess.env.TOKEN}`])
```